src/03_modules.py

```diff
@@ -10,22 +10,28 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print("This is the name of the script: ", sys.argv[0])
+print("Number of arguments: ", len(sys.argv))
+print("the arguments are: ", str(sys.argv), "argv2")
```

Comment:
This is a way to see all of the args, for sure, but the prompt is to print out the command line arguments one per line :)

If there's more than one, sys.argv will be an array, and you can iterate over it to print one per line, something like:

`for argument in sys.argv:`
    `print(argument)`